docker/colony-cdapp-dev-env-auth-proxy

```diff
@@ -1,6 +1,6 @@
 FROM colony-cdapp-dev-env/base:latest
 
-ENV AUTH_PROXY_HASH=f1d5e35271358946ec65e9a0f7d27fa66c2b878a
+ENV AUTH_PROXY_HASH=b83e72fd73bc676fc78872d2a646c140e521c0b3
```

Comment:
This needs to be adjusted once https://github.com/JoinColony/colonyCDappAuthProxy/pull/25 and https://github.com/JoinColony/colonyCDappAuthProxy/pull/26/files are merged